README.rst

```diff
@@ -43,13 +43,31 @@ libraries in the ``pep0`` directory.
 Checking PEP formatting and rendering
 =====================================
 
-Do not commit changes with bad formatting.  To check the formatting of
-a PEP, use the Makefile.  In particular, to generate HTML for PEP 9999,
-your source code should be in ``pep-9999.rst`` and the HTML will be
-generated to ``pep-9999.html`` by the command ``make pep-9999.html``.
-The default Make target generates HTML for all PEPs.
-
-If you don't have Make, use the ``pep2html.py`` script directly.
+Please don't commit changes with reStructuredText syntax errors that cause PEP
+generation to fail, or result in major rendering defects relative to what you
+intend. To check building the HTML output for your PEP (for example, PEP 12)
+using the current default docutils-based system, run the ``pep2html.py`` script
+with your PEP source file as its argument (e.g. for PEP 12,
+``python -X dev pep2html.py pep-0012.rst``).
+If you're on a system with ``make``, you can instead simply execute, e.g.,
+``make pep-0012.rst``.
```

Comment:
Sorry if it wasn't clear from the above, but I did eliminate it here. I always run development code locally and on CIs with it as a best practice to catch any warnings and other issues, but in this case since the focus is on correctness of the reST source, not the code, and reST warnings are handled separately, there isn't the same benefit to recommending it here.

It would be a good idea to use it on our CIs, though, at least once the docs are building cleanly with no warnings using the new rendering system, as on other Sphinx-based projects its useful to catch issues early and potential problems that would otherwise pass silently. But that's a separate issue.